docs/tutorial/pypi-package.rst

```diff
@@ -35,6 +35,14 @@ Next, create a file called ``rockcraft.yaml`` with the following contents:
     :caption: rockcraft.yaml
     :language: yaml
 
+.. note::
+
+    This example uses ``python`` plugin along ``python-packages``, so ``pip`` is
+    invoked implicitly to install such packages. `PEP 668`_ changes the
+    behaviour of Python package managers like ``pip`` by introducing a special
+    marker (``EXTERNALLY-MANAGED``) that prevents ``pip`` from installing
```

Comment:
Would this be more precise?

```suggestion
    environment variable (``EXTERNALLY-MANAGED``) that prevents ``pip`` from installing
```

Reply:
It's not an environment variable - it's a file. Personally I think it's fine to keep it as the vague "marker"